.vib/magento/goss/magento.yaml

```diff
@@ -0,0 +1,44 @@
+group:
```

Comment:
Reviewing possibles check for the asset itself, I think checking the magento cli works (beyond the version check) could be useful. I think we can test the modules feature by enabling one of them, WDYT? An example of this could be:

```
$ magento module:enable Magento_Store
```